.github/workflows/e2e.yml

```diff
@@ -0,0 +1,122 @@
+name: e2e
+
+on: [push]
+env:
+  REGISTRY: 'ghcr.io/dell/csi-baremetal'
+  go_version: '1.16.8'
+  golangci_version: '1.37.0'
+  helm_version: '3.7.1'
+  kubectl_version: 'v1.22.2'
+jobs:
+  e2e:
+    runs-on: ubuntu-20.04
+    if: "contains(github.event.head_commit.message, '[e2e]')"
```

Comment:
Is it hard to change from specific commit message to posting some comment in PR?

Reply:
Yes, it is no special action or keyword for it and we need to parse message and args manually in CI. It is better to do it in the separate task

Reply:
Yes, it is no special action or keyword for this and we need to parse a massage and args manually in CI. It is better to do it in the separate task

Reply:
UPD: will be done here

Reply:
...Also in the proposal https://github.com/dell/csi-baremetal/pull/605 (which I relied on), it isn't written what behavior is expected from this action - except for the most obvious - the successful launch of tests